<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>581</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(124, 0, 0);
border-radius: 30px;</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Program Studi</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Script MT Bold</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Data Dosen</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="optLaki">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>100</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Laki-laki</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cboProdi">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>IND</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TIF</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PET</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nama Lengkap</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClear">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>120</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#btnClear{
	background-color: rgb(0, 255, 0);
}</string>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="optPerempuan">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>100</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Perempuan</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>50</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>AZIZ MAULANA</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNIDN">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>NIDN</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>80</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>200511084</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNama">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSimpan">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>120</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#btnSimpan{
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 255);
}</string>
     </property>
     <property name="text">
      <string>Simpan</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnHapus">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>120</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#btnHapus{
	color: rgb(255, 255, 255);
	background-color: rgb(255, 0, 0);
}
</string>
     </property>
     <property name="text">
      <string>Hapus</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Jenis Kelamin</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="gridDosen">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>541</width>
       <height>192</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
gridline-color:  rgb(124, 0, 0);</string>
     </property>
     <property name="rowCount">
      <number>10</number>
     </property>
     <property name="columnCount">
      <number>5</number>
     </property>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
    <widget class="QPushButton" name="btnCari">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>50</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: gold;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>cari</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_6">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>450</y>
      <width>621</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(124, 0, 0);
border-radius: 70px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>350</y>
      <width>621</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(124, 0, 0);
border-radius: 70px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>621</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(124, 0, 0);
border-radius: 70px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_7">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>220</y>
      <width>141</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(124, 0, 0);
border-radius: 70px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_8">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>141</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(124, 0, 0);
border-radius: 70px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_9">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>141</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(124, 0, 0);
border-radius: 70px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_10">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>210</y>
      <width>141</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(124, 0, 0);
border-radius: 70px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_11">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>200</y>
      <width>141</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(124, 0, 0);
border-radius: 70px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_12">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>141</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(124, 0, 0);
border-radius: 70px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_13">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>180</y>
      <width>141</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(124, 0, 0);
border-radius: 70px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_14">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>180</y>
      <width>141</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(124, 0, 0);
border-radius: 70px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_15">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>150</y>
      <width>141</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(124, 0, 0);
border-radius: 70px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_16">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>170</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 40px;
 border-image: url(&quot;ui/um1.png&quot;);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>151</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 75px;
border-image: url(&quot;ui/guru.png&quot;) 0 0 0 0 stetch stretch;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>200</y>
      <width>151</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 75px;
border-image: url(&quot;ui/guru2.png&quot;) 0 0 0 0 stetch stretch;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <zorder>frame_15</zorder>
   <zorder>frame_14</zorder>
   <zorder>frame_13</zorder>
   <zorder>frame_12</zorder>
   <zorder>frame_11</zorder>
   <zorder>frame_10</zorder>
   <zorder>frame_9</zorder>
   <zorder>frame_8</zorder>
   <zorder>frame_7</zorder>
   <zorder>frame_4</zorder>
   <zorder>frame_5</zorder>
   <zorder>frame_6</zorder>
   <zorder>frame</zorder>
   <zorder>frame_16</zorder>
   <zorder>frame_2</zorder>
   <zorder>frame_3</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
